Jenkinsfile.nightly

```diff
@@ -99,19 +99,6 @@ pipeline {
                 }
             }
         }
-
```

Comment:
@aanm notice that the focus here is `K8s*` not `Nightly*`. What Eloy is saying is that since we have [a separate job](https://jenkins.cilium.io/view/Cilium-master/job/cilium-master-K8s-all/) which runs all K8s tests (with prefix `K8s*`) against all K8s versions daily, including 1.8, so we don't need to run it in the Nightly job as well. I agree with @eloycoto .